code/src/index.css

```diff
@@ -6,8 +6,232 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
-
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+/* The page´s mothership */
+div#root {
+  overflow-x: hidden;
+  margin: auto;
+  padding: 10px;
+  box-sizing: border-box;
+  overflow-x: hidden;
+  background-color: rgb(30, 30, 30);
+}
+
+/* HEADER */
+header {
+  width: 100%;
+  overflow-x: hidden;
+}
+.header-line {
+  border-bottom: 1.5px solid #d8d8d8;
+  -webkit-filter: opacity(12%);
+  filter: opacity(15%);
+  margin: auto;
+  width: 100%;
+}
+/* New albums & singels text */
+h1 {
+  color: #fbfbfb;
+  font-size: 24px;
+  padding: 10px;
+  margin: 15px 12px 0;
+  text-align: center;
+}
+/* Album title */
+h2 {
+  color: #ffffff;
+  font-size: 14px;
+  font-family: Helvetica;
+  margin: 35px 0 5px;
+  font-weight: 650;
+}
+/* Artist name */
+h3 {
+  color: #ffffff;
+  font-size: 14px;
+  font-family: Helvetica;
+  margin: 7px 0 5px;
+  font-weight: 650;
+  
+}
+a {
+  text-decoration: none;
+  color: #a0a0a0;
+  font-family: Helvetica;
+  font-size: 14px;
+  font-weight: 650;
+ margin-bottom: 35px;
+  margin-top: -87px;
+}
+
+
+/* ALBUM  */
```

Comment:
Good naming and delineation between sections of styling.  Makes it easy to understand which styling applies to which elements/sections on the page.